src/Microsoft.DotNet.XHarness.TestRunners.Common/ApplicationOptions.cs

```diff
@@ -99,30 +68,26 @@ public ApplicationOptions()
 
             var os = new OptionSet() {
                 { "autoexit", "Exit application once the test run has completed.", v => TerminateAfterExecution = true },
-                { "autostart", "If the app should automatically start running the tests.", v => AutoStart = true },
```

Comment:
This is wrong. Yes, it is not currently used, the issue is that if we want to support forms or do any type of IDE integration we want to be able to tell the application not to auto start running all the tests. If we remove this from the option set we will have to add it leter.